src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 101;
+    private final ColorSupplier supplier = new ColorSupplier();
```

Comment:
It's better to be more specific in naming, **supplier** is too abstract, colorSupplier would be a better choice